code/main/services/networkResponse.brs

```diff
@@ -29,18 +29,12 @@ function _adb_NetworkResponse(responseCode as integer, responseBody as string) a
 
         isSuccessful: function() as boolean
             ' https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses
-            if m._responseCode >= 200 and m._responseCode < 300 then
-                return true
-            end if
-            return false
+            return m._responseCode >= 200 and m._responseCode < 300
         end function,
 
         isRecoverable: function() as boolean
             ' RECOVERABLE_ERROR_CODES = [408, 504, 503]
-            if m._responseCode = 408 or m._responseCode = 504 or m._responseCode = 503 then
-                return true
-            end if
-            return false
+            return  m._responseCode = 408 or m._responseCode = 504 or m._responseCode = 503
```

Comment:
for mobile the recoverable codes are: 408, 429, 502, 503, 504. Can we match them here?

Reply:
I copied those recoverable codes from the mobile core : https://github.com/adobe/aepsdk-core-ios/blob/eccca8b8cbc47a41edbc30df5ff6b398fb094091/AEPServices/Sources/network/NetworkServiceConstants.swift#L16

Where is the code to handle other recoverable codes?

Reply:
```502 error is usually not something you can fix, but requires a fix by the web server or the proxies you are trying to get access through.```

It seems that we are able to include `429`, but not `502`.